src/test/scala/temple/builder/DatabaseBuilderTestData.scala

```diff
@@ -73,4 +75,102 @@ object DatabaseBuilderTestData {
         ),
       ),
     )
+
+  val sampleInsertStatement: Statement = Insert(
+    "test-service",
```

Comment:
I wonder if this should be `test_service`? Might need to fix our `snakeCase` function